extensions/podman/src/extension.ts

```diff
@@ -1802,16 +1804,26 @@ function sendTelemetryRecords(
         console.trace('unable to check from which path podman is coming', error);
       }
 
-      // add qemu version
+      // add krunkit version
       try {
-        const qemuVersion = await qemuHelper.getQemuVersion(qemuPath);
-        if (qemuPath) {
-          telemetryRecords.qemuPath = qemuPath;
+        const krunkitVersion = await krunkitHelper.getKrunkitVersion(krunkitPath);
+        if (krunkitPath) {
+          telemetryRecords.krunkitPath = krunkitPath;
```

Comment:
I don't understand why it's call krunkit path and not podman path ?

Reply:
It is the same path as podman (as it is the path where all the binaries installed by the podman installer), but it is intended to be used to find the krunkit binary